ui/static/ui/js/listing/listing.jsx

```diff
@@ -9,6 +9,7 @@ define('listing',
     'use strict';
 
     var EMAIL_EXTENSION = '@mit.edu';
+    var lastSelectedResourceId;
```

Comment:
After `closeLearningResourcePanel` is moved inside the class you should be able to just use `this.state.currentResourceId` which already exists